pkgs/data/fonts/nerd-fonts/convert-license.nix

```diff
@@ -0,0 +1,17 @@
+{ lib }:
+with lib.licenses;
+licenseString:
+{
+  "Bitstream-Vera AND MIT" = [
+    bitstreamVera
+    mit
+  ];
+  "LicenseRef-Monofur" = free; # upstream `src/unpatched-fonts/Monofur/LICENSE.txt`
+  "LicenseRef-UbuntuFont" = ufl;
+  "LicenseRef-VicFieger" = free; # upstream `src/unpatched-fonts/HeavyData/Vic Fieger License.txt`
+  "MIT OR OFL-1.1-no-RFN" = mit;
+  "OFL-1.1-RFN" = ofl;
+  "OFL-1.1-no-RFN or LGPL-2.1-only" = ofl;
+  "OFL-1.1-no-RFN" = ofl;
+}
+.${licenseString} or licenseString # Others simply use single SPDX license IDs.
```

Comment:
And here use `getAttr` or something alike. Plus, I'm not sure I like that `or licenseString`, I think an evaluation failure should be thrown when a license existing in upstream's list is not mapped to something in this table.